sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -458,6 +464,69 @@ def _is_live(self, action):
         except TypeError:
             pass
 
+    def _settle_message(
+            self,
+            settle_operation,
+            dead_letter_details=None
```

Comment:
isn't there dead_letter_reason as well?  Or are we leaving that out until we mitigate the dead letter receiver-link settlement issue.

Reply:
misread